src/main/java/org/apache/commons/imaging/formats/jpeg/decoder/JpegDecoder.java

```diff
@@ -242,43 +242,41 @@ public boolean visitSegment(final int marker, final byte[] markerBytes,
         } else if (marker == JpegConstants.DQT_MARKER) {
             final DqtSegment dqtSegment = new DqtSegment(marker, segmentData);
             for (final QuantizationTable element : dqtSegment.quantizationTables) {
-                final DqtSegment.QuantizationTable table = element;
```

Comment:
Or we could rename `element` to `table`? WDYT @arturobernalg ? This way the "Invalid quantization table identifier" message would clearly reflect that the identifier is from `table` (as in the variable name too).

Table is also a common term in many image processing documents (I guess that's why the author of the code created the alias-variable).

Reply:
HI @kinow 
I am agree with `table` name. Seem more logical.